pypeit/spectrographs/keck_deimos.py

```diff
@@ -403,21 +453,26 @@ def get_rawimage(self, raw_file, det):
 
         """
         # Check for file; allow for extra .gz, etc. suffix
+        # TODO: Why not use os.path.isfile?
```

Comment:
if `isfile` can handle wild cards, we should use it.

Reply:
My point was more that `get_rawimage` should, in my opinion, only ever take a filename, and it should fault if that filename doesn't match a file that exists.  If you don't agree, I can leave it as is.  Do you know of a case where this functionality is needed?

Reply:
I have known people (including myself) to uncompress files
to speed things up after generating the PypeIt file.

That said, it's not a use case we need to support.